plugins/invisible-typing/src/Settings.tsx

```diff
@@ -16,7 +16,7 @@ export function Settings(): React.ReactElement {
       </SwitchItem>
       <SwitchItem
         {...util.useSetting(cfg, "invisible", true)}
-        note="When enabled, invisibility applies to all channels. When disabled, invisibility is channel based.">
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Clarified setting description for better user understanding.

The updated note provides clearer information on how the global invisibility setting interacts with channel-specific settings, enhancing user experience.

```suggestion
note="Invisibility can be applied globally to all channels or individually per channel. When enabled globally, it sets the default invisibility state for channels, unless specified otherwise at the channel level.">
```